.changeset/slow-maps-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
```

Comment:
Hi @Skyflash 
Could you change "minor" to "patch"?

Reply:
Ok. I have to make a new PR or i can modify this?

Reply:
Hey @Skyflash You can delete the old one